src/plugins/ItemSearchPlugin/LanguageItemSearch.ts

```diff
@@ -0,0 +1,13 @@
+import ItemSearcher from '@/data-access/ItemSearcher';
+import {
+	inject,
+	InjectionKey,
+} from 'vue';
+
+export const LanguageItemSearchKey: InjectionKey<ItemSearcher> = Symbol( 'LanguageItemSearch' );
+
+export function useLanguageItemSearch(): ItemSearcher {
+	return inject( LanguageItemSearchKey, () => {
+		throw new Error( 'No ItemSearcher provided!' );
```

Comment:
Hm, wouldn’t it be more helpful if this message still said “No **Language**ItemSearcher provided”, even if we’re using the more general `ItemSearcher` as the return type?

Reply:
Done